app/src/common/shared/org/mozilla/vrbrowser/VRBrowserApplication.java

```diff
@@ -26,6 +27,7 @@ public void onCreate() {
         mPlaces = new Places(this);
 
         TelemetryWrapper.init(this);
+        GleanMetricsService.init(this);
```

Comment:
This could just be the `Glean.INSTANCE.initialize(...)`...

Reply:
Same here.